OpenSSL requirement for installing PG15 on windows. (#1942)
timescale/docs: install/installation-windows.md

[Install from source](/install/latest/self-hosted/installation-source/) instead.
</highlight>

## Prerequisites

> Make sure OpenSSL 1.1.1 is installed on the system if installing
> PostgreSQL version 15.1.1 or above.

Reviewer: ```suggestion
To install PostgreSQL version 15 or later, make sure you have:

* Installed OpenSSL 1.1.1 or later
```

What do you think?

    click `Edit...`. In the `Edit environment variable` dialog, click `New` and
    type the path to your PostgreSQL `pg_config` file. It should
    be `C:\Program Files\PostgreSQL\14\bin\`. Click `OK` to save your changes.
1.  If installing PostgreSQL version 15.1.1 or above, make sure OpenSSL 1.1.1 is

>     installed on the system.

Reviewer: This is actually a prerequisite for the procedure, not a step. The test is that a step always starts with a verb (like "Select the option") or a location (like "In the dialog, select...").  Pre-reqs are listed above the procedure with a stem sentence like "Before you start, make sure you ...". You can see an example here: https://docs.timescale.com/install/latest/self-hosted/installation-source/

Reviewer: Also, more about writing processes and procedures here: https://docs.timescale.com/timescaledb/latest/contribute-to-docs/#processes-and-procedures
